openpype/hosts/maya/api/workfile_template_builder.py

```diff
@@ -244,13 +233,36 @@ def get_placeholder_options(self, options=None):
         return self.get_load_plugin_options(options)
 
     def post_placeholder_process(self, placeholder, failed):
-        """Hide placeholder, add them to placeholder set
-        """
+        """Hide placeholder, add them to placeholder set"""
         node = placeholder.scene_identifier
 
         cmds.sets(node, addElement=PLACEHOLDER_SET)
         cmds.hide(node)
         cmds.setAttr(node + ".hiddenInOutliner", True)
+        model = cmds.ls(assemblies=True)
+
+        filtered_model = []
+        exclude_list = ["persp", "top", "front", "side", "root", "wip"]
```

Comment:
"root" and "wip" I assume are very likely hardcoded examples from your workflow?

Reply:
yes they are